pkg/minishift/openshift/service.go

```diff
@@ -67,38 +107,42 @@ func GetServiceURL(service, namespace string, https bool) (string, error) {
 		return "", errors.New(fmt.Sprintf("No service with name '%s' defined in namespace '%s'", service, namespace))
 	}
 
-	url := strings.Split(byteArrayToString(cmdOut), "\n")[1] // second element contain actual URL content
+	url := strings.Split(string(cmdOut), "\n")[1] // second element contain actual URL content
 	return urlScheme + url, nil
 }
 
-// Get the available routes to user
-func GetServiceURLs(serviceListNamespace string) ([]ServiceURL, error) {
-	var serviceURLs []ServiceURL
+// GetServiceSpecs takes Namespace string and return route/nodeport/name/weight in a ServiceSpec structure
```

Comment:
technically it takes a list/slice of namespaces and returns a list/slice of ServiceSpec instances